easybuild/tools/robot.py

```diff
@@ -359,6 +359,9 @@ def search_easyconfigs(query, short=False, filename_only=False, terse=False):
     search_path = build_option('robot_path')
     if not search_path:
         search_path = [os.getcwd()]
+    extra_search_paths = build_option('search_paths')
```

Comment:
Resolved in https://github.com/easybuilders/easybuild-framework/pull/2255/commits/58262d770d8c3ce0887f128e3316688613e1d203